pallets/pips/src/lib.rs

```diff
@@ -1348,15 +1364,54 @@ impl<T: Trait> Module<T> {
         Self::prune_data(did, id, new_state, Self::prune_historical_pips());
     }
 
-    fn schedule_pip_for_execution(did: IdentityId, id: PipId) {
-        // Set the default enactment period and move it to `Scheduled`
-        let curr_block_number = <system::Module<T>>::block_number();
-        let executed_at = curr_block_number + Self::default_enactment_period();
-
+    // TODO: the `maybe_at` argument is only required until `schedule::Named::reschedule_named` is
+    // released.
+    fn schedule_pip_for_execution(
+        did: IdentityId,
+        id: PipId,
+        maybe_at: MaybeBlock<T::BlockNumber>,
+    ) {
+        // Set the enactment period and move it to `Scheduled`
+        let at = match maybe_at {
+            MaybeBlock::Some(at) => at,
+            MaybeBlock::None => {
+                <system::Module<T>>::block_number() + Self::default_enactment_period()
+            }
+        };
```

Comment:
```suggestion
        let at = at
            .unwrap_or_else(|| <system::Module<T>>::block_number() + Self::default_enactment_period());
```
The `MaybeBlock` stuff is just a hack to make the polkadot UI happy. We can use `Option<T::BlockNumber>` internally.